etc/dnf/aliases.d/zypper.conf

```diff
@@ -4,4 +4,5 @@ dist-upgrade = distro-sync
 ref = makecache
 refresh = makecache
 ri = reinstall
+wp = provides
```

Comment:
Is it meant to be "provides" or "whatprovides"?

Reply:
It is supposed to be `whatprovides`, though that goes to the same subcommand. Honestly, I think this one probably belongs in the code rather than in here.

Reply:
Oh, so we have the `provides` command, which also has a `whatprovides` alias, which both list packages that provide the argument. Then we have the `--provides` and `--whatprovides` options of the `repoquery` command, where `--whatprovides` does the same, but `-provides` lists the provides of the packages that match the argument... Nice.

So the reasoning to have this as an alias in the code is that it's universally useful, not really related to zypper?

Reply:
Yep.

Reply:
I've gone ahead and made this a documented alias.